solutions_python_book/lazy_student/lazy_student.py

```diff
@@ -0,0 +1,85 @@
+import sys
+
+
+def main():
+    file_home_work_name = "home_work.txt"
+    file_solution_name = "solutions.txt"
+    # file_home_work_name = sys.argv[1]
+    # file_solution_name = sys.argv[2]
+
+    try:
+        with open(file_home_work_name, "r") as f_read:
+            all_exercises = f_read.readlines()
+
+            with open(file_solution_name, "w") as f_write:
+                for line in all_exercises:
+                    if check_line(line):
+                        exercise_lst = line[:-1].split(" ")
+                        try:
+                            result = str(simple_calc(exercise_lst))
+                            f_write.write(result + "\n")
+                        except ZeroDivisionError as e:
+                            f_write.write("The exercise is not valid, you cant divide by zero: %s" % line)
+                    else:
+                        f_write.write("The exercise was not written write!: %s" % line)
+    except IOError as e:
+        print("The path of the home work file is not correct try again", e)
+
+
+def check_line(line):
+    """
+    Have to be in this format: "num,space,operator,space,num,\n"
+    :param line: string exercise
+    :return: boolean value.
+    """
+    # check for "\n"
+
+    if line[-1] != "\n":
+        return False
+    line = line[:-1]
+
+    # check if there is an operator, if so what it is and what's its index.
```

Comment:
It's guaranteed that spaces will separate the operator and the numbers.
So it's much simpler to use `split`.